viewing_party/party.py

```diff
@@ -0,0 +1,150 @@
+#wave 1
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        movie_dict = {
+            "title" : movie_title,
+            "genre" : genre,
+            "rating" : rating,
+        }
+        return movie_dict
+    else:
+        return None
+
+#wave 2
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    if movie in user_data["watchlist"]:
+        print("This movie is already in the watchlist!")
+    else:     
+        user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title == movie["title"]:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
```

Comment:
this is a very redundant suggestion since this is only one line of code and thus works just fine, but where have we seen this before? maybe we could use `add_to_watched` to do this for us. Again, suuuper unnecessary change to make in the scheme of things, so think of this as just something to consider for next time.

```suggestion
            add_to_watched(movie)
```